code/modules/clothing/masks/muzzle.dm

```diff
@@ -25,3 +25,52 @@
 	clothing_flags = MASKINTERNALS | BLOCKS_SPEECH
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 100, FIRE = 0, ACID = 0)
 	equip_delay_other = 25 // my sprite has 4 straps, a-la a head harness. takes a while to equip, longer than a muzzle
+
+/obj/item/clothing/mask/muzzle/tape
```

Comment:
I couldn't figure out a way to get the attacker's targeted body zone since `target_zone` didn't work but I thought it would be fine to just have it be any target since you won't use it on people for any other reason, like cuffs.